src/assets/stylesheets/index.scss

```diff
@@ -356,3 +356,45 @@ body {
   }
 }
 
+#mc_embed_signup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+#mc_embed_signup_scroll {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+:local(.subscribeBar) {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+:local(.subscribe-text) {
+  font-size: 14pt;
+  font-weight: lighter;
+  margin: 0px 0px 8px 0px;
+}
+
+:local(.subscribe-button) {
+  @extend %action-button-secondary;
+  height: 40px;
+}
+
+:local(.email-input) {
+  height: 40px;
+  border-radius: 24px;
+  margin: 0px 16px 0px 0px;
+  width: 24em;
+  text-align: center;
+}
```

Comment:
Is there a reason this input deviates from our existing design for inputs? I would try to use our existing classes to maintain design consistency or at least try to match the style.
In particular, I'm talking about the shape of the input, the border around it and the font family, font alignment and font size of the input text:
![image](https://user-images.githubusercontent.com/79419/78591242-ffc5c500-77f7-11ea-85f9-3c7a2df7b830.png)
![image](https://user-images.githubusercontent.com/79419/78591276-0ce2b400-77f8-11ea-8cf1-87f6a54dace7.png)

Reply:
Good point. I started to `@extend %input-field` but then removed the in-page sign up entirely.